src/components/CommerceReviewsList.js

```diff
@@ -30,7 +32,7 @@ class CommerceReviewsList extends Component {
   render() {
     return this.props.loading ? (
       <Spinner />
-    ) : this.props.commerceReviews.length > 0 ? (
+    ) : this.props.commerceReviews && this.props.commerceReviews.length > 0 ? (
```

Comment:
Por que la doble validacion?

Reply:
No me preguntes por qué pero a veces me tiraba error esa línea diciéndome que `this.props.commerceReviews` era `undefined` (no se por qué si el reducer la inicializa como array vacío)

Reply:
jajajaj y fijate, en algun lado la debes estar sobreescribiendo.

Reply:
Ya he buscado mijo, si no hay muchas propiedades commerceReviews como pa no fijarse

Reply:
jajaja na pero digo, capaz en alguna parte el reducer sin querer la borras o algo asi.